lib/optimizely/decision_service.rb

```diff
@@ -417,6 +417,28 @@ def get_forced_variation(project_config, experiment_key, user_id)
       [variation, decide_reasons]
     end
 
+    def validated_forced_decision(context, user_context)
+      decision = user_context.find_forced_decision(context)
```

Comment:
Remove `find_forced_decision` and use `get_forced_decision`. Since we remove sanity checking, they're now identical.